lib/Agama/agama_base.pm

```diff
@@ -0,0 +1,20 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: First installation using D-Installer current CLI (only for development purpose)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Agama::agama_base;
```

Comment:
I know there are some that start by capital letter, but for this one, it is just a logical separation, I think it should `agama::agama_base`. Like "containers".

Reply:
btw it should be under `lib/yam/agama` for each main folder in the repo, schedule, lib, tests, etc, we should have our yam folder and put there our best stuff.